.github/scripts/assignPullRequestReviewer.js

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+
+const {
+  teams: { translationReviewers },
+} = require("./support/resources");
+
+/** @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments */
+module.exports = async ({ github, context }) => {
+  const { repo, owner } = context.repo;
+
+  const payload = /** @type {import('@octokit/webhooks-types').PullRequestEvent} */ (context.payload);
+  const {
+    pull_request: { number: pull_number },
+  } = payload;
+
+  const reviewers = (
+    await github.rest.teams.listMembersInOrg({
+      org: owner,
+      team_slug: translationReviewers,
+    })
+  ).data.map((member) => member.login);
+
+  const { data: reviews } = await github.rest.pulls.listReviews({ owner, repo, pull_number });
+
+  // passes if there was a previous approval from an admin
+  reviews.forEach((review) => {
```

Comment:
This could be simplified by using `some`:

```ts
if (
  reviews.some((review) =>
    review.state === "APPROVED" && 
    review.user?.login &&
    reviewers.includes(review.user.login)
  )
) {
  process.exit(0);
}
```